possel/application.py

```diff
@@ -82,17 +82,19 @@ def get_etc_file(filename):
 
 def get_arg_parser():
```

Comment:
I'd recommend passing in a dict of defaults to the config object rather than setting them on argparse in that case